scripts/contributing.js

```diff
@@ -0,0 +1,36 @@
+/* global jQuery*/
+
+(function($){
+    var githubIssuesEndpoint = 'https://api.github.com/repos/babel/babel/issues';
```

Comment:
>  Note that: ... I don't know if we want PR there.

Yeeeeeeep. That was an oversight. I even made a note to check for that. I'll address that in a future commit.

>  Do you know if there quotas on API calls?

Great question. After looking into it there's a 60 request/hour/IP limit for unauthenticated requests according to the [Github docs](https://developer.github.com/v3/#rate-limiting). There's a caching header on the response of 60 seconds, so theoretically we could do a single call as much as we want. Since I've used 2 here, there's a _possible_ issue if someone were to refresh every minute for long enough, but that doesn't seem practical. During my testing, I wasn't able to use more than 20/hour. 

I guess I don't see this as a practical enough issue _as is_ - though this might be a reason to split up repos across multiple pages rather than aggregating all issues for all repos on the one page.